spec/wordlist_repo_spec.rb

```diff
@@ -14,14 +14,14 @@
   end
 
   describe "#download" do
-    let(:repo_path) { File.join(Dir.pwd, "") }
+    let(:repo_path) { "/does/not/exist/" }
     let(:url)       { "https://www.example.com" }
 
     context "when git clone succeed" do
       it "must return new wordlist repo" do
         expect(described_class).to receive(:system).with('git', 'clone', '--depth', '1', '--', url, repo_path).and_return(true)
 
-        expect(described_class.download(url)).to be_kind_of(Ronin::Wordlists::WordlistRepo)
+        expect(described_class.download(url, repo_path)).to be_kind_of(Ronin::Wordlists::WordlistRepo)
```

Comment:
This should be a separate `context` edge-case where the destination path is given. By default if the destination path is not given, it will default to the current directory. Feel free to simply copy/paste this spec, but put it within a `context` with the `let(:repo_path)`.